triton-vm/src/table/extension_table.rs

```diff
@@ -204,11 +213,8 @@ pub trait Quotientable: ExtensionTable + Evaluable {
             .par_iter()
             .enumerate()
             .map(|(fri_dom_i, &z_inv)| {
-                let row = codewords
-                    .iter()
-                    .map(|codeword| codeword[fri_dom_i])
-                    .collect_vec();
-                let evaluated_bcs = self.evaluate_initial_constraints(&row, challenges);
+                let row: &[XFieldElement] = &transposed_codewords[fri_dom_i];
```

Comment:
Is the type not automatically inferred?

Reply:
Yes. But this type was made explicit to ensure that no new allocation was made inside the busy-loop. It's important that memory is not allocated in there if that can be avoided.